packages/dd-trace/test/telemetry/index.spec.js

```diff
@@ -9,6 +9,8 @@ const { once } = require('events')
 const { storage } = require('../../../datadog-core')
 const os = require('os')
 
+const HEARTBEAT_INTERVAL = 60 // the default should be 60000, set it to 60 so that tests complete faster
```

Comment:
For tests like these it's a good practice to try to use artificial clocks. If you search the codebase for `clock.tick()` you'll see it in action. That function basically makes the process think that the provided amount of time has passed. That way you can run `clock.tick(60000)` and have the 60 second timer fire. Even though this one test is only taking 60ms, it can really add up if other tests also start taking additional 60ms to run.

Reply:
@tlhunter and I have decided it's too much work right now, so I added a TODO comment for later